app/views/content/train-to-be-a-teacher/qualifications-you-need-to-teach.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: "What qualifications do I need to be a teacher?"
 heading: "What qualifications do I need to be a teacher?"
-subcategory: Qualifications and experience
+subcategory: Qualifications
```

Comment:
I think this looks OK on the category page when you can see what is beneath it - is it descriptive enough in the dropdown menu though?
![image](https://github.com/user-attachments/assets/c9b57f11-3fe6-4b49-a7c4-9379f92e4bef)

Reply:
I think it's quite difficult to be more descriptive as some of the content is around qualifications you need to be a teacher, vs qualifications you get through training. Given this heading is under teacher training, I think that helps to add a bit more context. I'll also check if we have a ticket in the backlog to revamp the qualifications content, as that would help in terms of more descriptive navigation.

Reply:
Agreed to leave